coyote/views.py

```diff
@@ -24,6 +24,11 @@ def get_domain(url):
 
 
 def home(request):
+    """
+    it takes necessary data from frontend request and sent it to hackerearth API
+    if the code doesnt compiles, then it takes necessary keyword from error messages 
```

Comment:
`if the code doesnt compiles` can be converted to `if the code doesn't compile`

`then it takes the necessary keyword` => `then it finds the necessary keyword`